package.json

```diff
@@ -18,7 +18,12 @@
     "dev:backend": "turbo run build --filter config --filter database --filter types && turbo run dev --filter backend --filter config --filter database",
     "lint": "turbo run lint",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "build:backend": "turbo run build --filter config --filter database --filter types",
```

Comment:
oh, this is good idea, we may also change dev:backend to run build:backend before run dev in the future to reduce duplicated code.